source/fundamentals/transactions.txt

```diff
@@ -163,86 +173,88 @@ order details. Suppose you organize the data updates as follows:
      - Record the purchase information
 
    * - ``customers``
-     - update
-     - Append the order id to associate it with the customer
+     - update or upsert
+     - Append the order ``_id`` to the document for a customer
```

Comment:
From my understanding, if a customer makes a purchase, the _id from the order document is also appended to the customer document in this collection, so that there is something like an "order history" for each customer, in addition to a running, chronological record of all orders